src/components/BooksHolder.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Books from './Books';
+
+const BooksHolder = () => {
+  const BOOKS = [
+    { title: 'Piesie', author: 'Wise', id: 1 },
+    { title: 'Bankudade', author: 'Omane', id: 2 },
+  ];
+  return (
+    <div>
+      {BOOKS.map((book) => (
+        <Books key={book.id} title={book.title} author={book.author} />
+      ))}
+    </div>
+  );
+};
```

Comment:
- [ ]  Your BooksHolder component should Renders a form by using the CreateNewBook component  which is AddBook component in your project.  I can see that you've imported your AddBook component in the App component, while it's supposed to be rendered in the BookHolder component.